### ISSUE TITLE
Need more nuanced "organization-project" relationships

### ISSUE THREAD

**Issue by jberkus** [cornelius/open-source-governance#2]
```
# Name of the organization providing governance to the project
organization_name: ?
```

So there's a couple assumptions that this makes:

1. That there's only one organization per project;
2. That that organization supplies governance, and not just financial support.

Instead, we need a nested JSON structure here:

```
organization_1: { 
  organization_name:
  organization_type:
  organization_relationship: [governance, financial support, other]
  ...
}
```

**Comment by ShaneCurcuru** [comment]
Yes, for my purposes (I hope to expand chooseafoundation.com by sharing some of the data here) this needs to be richer.  Or rather, this needs a better definition of "project" for the context here.

My starting concept is:
- Organization (usually a legal one)
- Project (has a website, semi-organized **community**, produces some content or code)
- Product (software product or possibly a widely-used website of data or resources)

Separately, I hope to build a reasonably complete data structure for any major **organizations** that support FOSS in my work.  Maybe one day I could have a semi-definitive URL for each organization that could be pointed to as just (name, relationship to this project).

Good questions!

**Comment by cornelius** [comment]
Good points. This is one of the more challenging parts.

@jberkus do you have examples in mind where a project is related to more than one organization?

For the scope of the this project about governance data I would probably leave out the products because that can be a lot and I would go with the assumption for now that if a project does multiple products they have very similar governance structure. I think that might work as a first approximation.

**Comment by jberkus** [comment]
@cornelius the first two that come to mind are Linux Kernel and PostgreSQL.  Both have multiple organizations supplying material support.

**Comment by cornelius** [comment]
@jberkus How would the values for the Linux Kernel and PostgreSQL look like then? I'll add the templates for the two project. Let's see how we can fill them.

**Comment by cornelius** [comment]
It looks like it would make most sense to separate the organization information from the project information. Having separate files and using references. There are the projects which have more than one organization and there are the organizations which have more than one project.

**Comment by jberkus** [comment]
Yes, just think of SPI, the Apache Foundation, or LF.

Now, how can we create non-conflicting IDs for the organizations?  So that we can reference them?

**Comment by cornelius** [comment]
I separated the info about organizations into separate files now. Types of relationships still have to be defined.
